.github/workflows/cifuzz.yml

```diff
@@ -1,104 +1,40 @@
 name: CIFuzz
 on:
   workflow_dispatch:
+  pull_request:
```

Comment:
no need to add pull request here, fuzzer-ci runs every 107 minutes, if doesn't need to run for every pull request as well

Reply:
You are right, I think I've added it to test the workflow. I'll remove it